vyper/semantics/analysis/module.py

```diff
@@ -409,6 +410,10 @@ def visit_InitializesDecl(self, node):
         module_info = get_expr_info(module_ref).module_info
         if module_info is None:
             raise StructureException("Not a module!", module_ref)
+        if is_stateless(module_info.module_node):
```

Comment:
maybe should we also have this restriction on `UsesDecl` -- @cyberthirst wdyt?

Reply:
sorry, missed the ping

i think that makes sense - `uses` indicates that we're accessing the used module's state - and thus the module should actually have state